lib/downloadresponse.php

```diff
@@ -30,7 +30,10 @@ public function __construct(IRequest $request, $user, $path) {
 		$this->user = $user;
 		$this->path = $path;
 		
-		$this->view = new View('/' . $user);
+		$data_dir = \OC::$server->getConfig()->getSystemValue('datadirectory', \OC::$SERVERROOT . '/data');
+		$fake_root = str_replace($data_dir, '', \OC::$server->getUserManager()->get($user)->getHome());
```

Comment:
hmmm .... I see this fix now popping up the second time in two different places.

This indicates that we are only fixing the symptom but not the root cause.

@blizzz @icewind1991 can I ask you to get an understanding of the situation?

@blizzz because of getHome which id afaik only used in the ldap home integration
@icewind1991 to get this properly integrated into code